static/templates/message_controls.hbs

```diff
@@ -1,27 +1,27 @@
 <div class="message_controls{{#status_message}} sender-status-controls{{/status_message}} no-select">
     {{#if msg/sent_by_me}}
-    <div class="edit_content"></div>
+    <div class="edit_content message_control_button"></div>
```

Comment:
So I just discovered reading your other PR that we have a different thing called `message-control-button` that is a compose box thing.  I think the name for this widget is correct, sp we should do a separate PR to rename the compose box button concept to something like `compose_control_button`.

Reply:
Did in #18154